Dockerfile

```
@@ -0,0 +1,17 @@
+FROM python:3.11-slim-buster as base
+RUN curl -sSL https://install.python-poetry.org | python3 -

```

When I run this, curl isn't found - the error actually gets swallowed but it leads to Poetry not being installed. Maybe we're building on different OS and the base image differs slightly, but I'd expect to need:
```suggestion
FROM python:3.11-slim-buster as base
RUN apt-get update && apt-get install curl -y
RUN curl -sSL https://install.python-poetry.org | python3 -
```